### DISCUSSION TITLE
[Feature] Auto add photos to a default Album on import/backup

### DISCUSSION THREAD

**Discussion by Alacard** [immich-app/immich#8965] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Would be nice that when setting a folder/location to back up, you able to add a default album for that location example.

Lets say i want to back up all my facebook pictures, when i select the facebook location/folder to backup there a option that allows me to select a "Album" that any/all photo's found at that location will auto be added to.

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#1678
